app/modules/disk_report/disk_report_model.php

```diff
@@ -72,7 +72,23 @@ public function get_disk_type()
 						".get_machine_group_filter('AND');
         return current($this->query($sql));
     }
-
+    
+    /**
+     * Get filesystem type statistics
+     * autor tuxudo
+     *
+     **/
+    public function get_volume_type()
+    {
+        $sql = "SELECT VolumeType as type, count(*) as count FROM diskreport
```

Comment:
There's two times `FROM diskreport` in the query, does that even work? And please fix the indenting.